packages/firestore/test/unit/api/field_value.test.ts

```diff
@@ -24,4 +25,12 @@ describe('FieldValue', () => {
     expectEqual(FieldValue.serverTimestamp(), FieldValue.serverTimestamp());
     expectNotEqual(FieldValue.delete(), FieldValue.serverTimestamp());
```

Comment:
Consider adding a test here as well for calling isEqual() with an object that is not an instanceof FieldValue (e.g. a string).

Reply:
Done

Reply:
Update: I reverted it since FieldPath.documentId() throws if I pass a FieldValue, and the FieldValue constructor is typed to only take FieldValues.